docs/tutorial.rst

```diff
@@ -250,3 +250,109 @@ Word Error Rate variants
+
+The BEER is useful to evaluate:
+
+*  the suitability of transcript files as input to a tagging system
+*  the performances of STT services on key entities depending on the contexts, for instance highlights and players names for sport events, 
+*  the performances of a list of entities automatically selected in the reference text by TF/IDF approach which intend to reflect how important a word is.
+
+An entity is a word or an ordered list of words including capital letters and punctuation. To calculate BEER, BenchmarkSTT needs a list of entities. It does not make this list for you. It is expected that the user will create the list outside of BSTT, manually or by using an NLP library to extract proper nouns from the reference. 
+
+BEER definition
+---------------
+
+The BEER is defined as the error rate per entity with a bag of words approach. In this approach the order of the entities in the documents does not affect the measure.
+
+.. math::
+
+   {WA}\_{BEER} \left (  entity \right ) = \frac{ \left | n_{hyp} - n_{ref}  \right |  }{n_{ref} }
+
+..
+   _multiple ..math:: directives to work around right alignment of formulas  
+
+.. math::
+
+   n_{ref}=\textrm{number of occurrences of entity in the reference document} 
+   
+   n_{hyp}=\textrm{number of occurrences of entity in the hypothesis document}
+
+The weighted averaged BEER of a set of entities e\ :sub:`1`, e\ :sub:`2` ... e\ :sub:`n` measure the global performances of the n entities, a weight w\ :sub:`n` is attributed to each entity. 
+
+.. math::
+   
+   WA\_BEER (e_1 ... e_N) =  w_1*BEER (e_1)\frac{L_1}{L} +... + w_N*BEER (e_N)\frac{L_N}{L}
+
+.. math::
+
+   {WA}\_{BEER} \left (  e_1, ... e_N \right ) = \frac{w_1* \left | n_{hyp_1} - n_{ref_1}  \right | + ... + w_n * \left | n_{hyp_n} - n_{ref_n}  \right | }{L}
+
+.. math::
+
+   L_1=\textrm{number of occurrences of entity 1 in the reference document}
+
+.. math::
+   L=L_1 + ... + L_N
+
+The weights being normalised by the tool
+
+.. math::
+   w_1 + ... + w_N=1
+
+
+Calculating BEER
+----------------
+
+BenchmarkSTT does not have a built-in list of entities. You must provide your own in a JSON input file defining the list of entities and the weight per entity. 
+
+The file has this structure: 
+
+   ``{ "entity1" : weight1, "entity2" : weight2, "entity3" : weight2 .. }``
+
+Let's create an example list. Save the below list as file ``entities.json``::
+
+   {"Theresa May" : 0.5, "Abigail" : 0.5, "EU": 0.75, "Griffin" : 0.5, "I" : 0.25}
+
+We'll also tell BenchmarkSTT to normalize the reference and hypothesis file by lowercasing both. We do this in the ``config.conf`` file: ::
```

Comment:
We'll also tell BenchmarkSTT to normalize the reference and hypothesis file but without lowercasing both. We do this in the ``config.conf`` file: ::